misc/pp_mixed_scenarios/post_process_mixed_scenarios.py

```diff
@@ -235,7 +242,7 @@ def worker(src: Path, dst: Path):
             for item in src.rglob("*"):  # Recursively copy all contents
                 relative_path = item.relative_to(src)
                 target_path = dst / relative_path
-
+                dst.mkdir(parents=True, exist_ok=True)
```

Comment:
❓ None of the tests exercise that - is it required? We still require at least one IU to be taken from the so called default scenario right? If a user provides an empty list we will take the whole thing, and if there is one then we'll at least copy that one.